src/main/java/com/shippingcostestimator/enterprise/dao/ShipmentSQLDAO.java

```diff
@@ -22,8 +22,7 @@ public Shipment findShipId(int id) {
 
     @Override
     public Shipment saveEstimate(Shipment shipment) {
-        Shipment createdShipment = shipmentRepository.save(shipment);
-        return createdShipment;
+        return shipmentRepository.save(shipment);
```

Comment:
Removed a redundant variable and declaration to improve tech debt!

Reply:
Do you have a citation for how this improves tech debt?  Which of the Seven Deadly Sins applies to this?